client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+import { TASK_WP_COURSES } from 'calypso/my-sites/customer-home/cards/constants';
+import coursesLogo from 'calypso/assets/images/customer-home/courses-logo.png';
+
+const WpCourses = () => {
+	return (
+		<Task
+			title="Introducing WordPress Courses"
```

Comment:
These user-viewed texts should be set up for localization ([example](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/customer-home/cards/tasks/renew/index.js#L41)).